### DISCUSSION TITLE
Recommended manufacturer for Harp PCBs (or device.stepperdriver specifically)

### DISCUSSION THREAD

**Discussion by NabbefeldG** [harp-tech/protocol#109] [Q&A] [answered]
Hi, 
we are looking at getting some of the device.stepperdriver boards manufactured. Using the "JLC Fabrication Toolkit" in KiCad I generated the bom, gerber files, etc. and tried to order these PCBs from JLCPCB. However, there are a couple of components that cannot be automatically matched. I can try to match those manually and we should be able to order them this way. But, I was wondering if these boards were designed with a specific Vendor in mind, with whom this ordering process might be easier?
Thank you in advance!

**Comment by filcarv** [comment, marked as answer]
Hi,

We didn't had any PCB(A) vendor in mind because we don't have a preferred one, we use several.

**Reply by NabbefeldG** [reply]
Ok, that's good to know. Thanks!
